src/components/FeeSummary/FeeSummary.vue

```diff
@@ -5,36 +5,46 @@
       :payURL="payApiUrl"
     />
 
-    <v-row class="mt-1">
-      <v-col cols="8" class="pr-2">
+    <v-row no-gutters>
+      <v-col v-if="isSummaryMode" class="pt-3 pr-3">
         <v-btn
-          id="save-resume-btn"
+          id="back-btn"
           large
           :loading="isLoading"
-          @click="emitAction(SummaryActions.SAVE_RESUME_LATER)"
+          @click="emitAction(SummaryActions.BACK)"
         >
-          <span>Save and Resume Later</span>
+          <span><v-icon>mdi-chevron-left</v-icon>Back</span>
+        </v-btn>
+      </v-col>
+      <v-col class="pt-3">
```

Comment:
This is suspicious. Why do you need to add top padding to both columns? This should be done at the row element.

On the other hand, I've found it necessary to add "mt-n1" for some buttons to line up with the other columns.

Reply:
This is to space the buttons vertically without having to write a row for every single button. 
Although it could be done our current requirements and implementation make it pretty easy to just let the vuetify grid handle this, and we apply these padding adjustments. 

![Screen Shot 2022-03-09 at 1 58 40 PM](https://user-images.githubusercontent.com/53542131/157543367-c289cd67-9f8d-4f34-8515-e7ea337837c0.png)

Reply:
Also allows us to throw an additional button in and it handles the space between.

![Screen Shot 2022-03-09 at 1 58 37 PM](https://user-images.githubusercontent.com/53542131/157543685-9e2c571d-a0b9-4801-8816-f7355d7d07c4.png)

Reply:
Good answers :)